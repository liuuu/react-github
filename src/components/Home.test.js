import React from "react";
import renderer from "react-test-renderer";
import Home from "./Home";

import { MemoryRouter } from "react-router-dom";

it("Home renders correctly", () => {
  const component = renderer.create(
    <MemoryRouter>
      <Home />
    </MemoryRouter>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
