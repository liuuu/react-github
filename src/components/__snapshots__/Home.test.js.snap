// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="home"
>
  <div
    className="home-header"
  >
    <h1>
      This is a Demo site
    </h1>
  </div>
  <div
    className="democontent"
  />
  <div>
    <button>
      <a
        href="all"
        onClick={[Function]}
      >
        browser the Popular Projects of github
      </a>
    </button>
  </div>
</div>
`;
